components/definitions/newTab.d.ts

```diff
@@ -77,8 +77,8 @@ declare namespace NewTab {
   }
 
   export interface PersistentState {
-    togetherPromptDismissed: boolean
-    togetherSupported: boolean
+    braveTalkPromptDismissed: boolean
+    braveTalkSupported: boolean
```

Comment:
This will bring the prompt back for users on Nightly channel who dismissed it, but that's totally ok by me!